tshirts.cpp

```diff
@@ -1,22 +1,25 @@
 #include <assert.h>
 #include <iostream>
 
-char size(int cms) {
+char getTShirtSizeName(int sizeIncms) {

```

Comment:
Nice renaming to `sizeIncms`
Got me thinking... Would it be `shoulderSizeIncms`?